src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/AddressSearchResource.kt

```diff
@@ -74,7 +74,7 @@ class AddressSearchResource(private val addressSearchService: AddressSearchServi
     ],
   )
   fun searchForAddresses(
-    @Size(min = 5, max = 100)
+    @Size(min = 4, max = 100, message = "Search query must be more than 3 and no more than 100")
```

Comment:
This has been added to pervent silly address look up's, It could be removed if need be.
If we keep it we should add validation in the front end to check for this?!